### ISSUE TITLE
DatePicker Keyboard Navigation.

### ISSUE THREAD

**Issue by sasi-kala** [valor-software/ng2-bootstrap#6720]
Keyboard Accessibility is missing.

When a user presses "Enter" or "Space" on the Input Box then the Calendar should open and all calendar days should be navigable using arrow keys.

**Comment by sagar-bakraniya-ifour** [comment]
I'm also facing same issue not able to navigate dates using keyboard and date picker is focusable only after navigating each interactive element on page
